plugins/modules/rulebook_activation.py

```diff
@@ -131,19 +142,15 @@
       - This parameter is supported in AAP 2.5 and onwards.
         If specified for AAP 2.4, value will be ignored.
     type: str
-    default: "error"
     choices: ["debug", "info", "error"]
   state:
     description:
       - Desired state of the resource.
     default: "present"
-    choices: ["present", "absent"]
+    choices: ["present", "absent", "enabled", "disabled"]
```

Comment:
Since we allow the edition of an activation, the behavior of the module changes significantly and we need to explain in detail the behavior. We have to state that "present" is equivalent to "enabled" unless future deprecated "enabled" param is defined (the default is True). Also we have to say that in "present", "enabled" and "disabled" states, the activation will be created if it not exists.